assets/shared/blocks/progress-bar/progress-bar.js

```diff
@@ -22,7 +22,7 @@ import { sprintf, _n } from '@wordpress/i18n';
  * @param {string}  props.lessonsCountClassName   Lessons count class name.
  * @param {string}  props.completedCountClassName Completed count class name.
  */
-const CourseProgress = ( {
+const ProgressBar = ( {
 	lessonsCount,
```

Comment:
@donnapep I understand the motivation behind not changing this block related progress bar stuff, so my issue here is that this specific component is the one that we are going to use on the pagination and we would need it to be renamed.

So the solution I propose here is to move this component to a new folder called components and import it from there, where it's used.